README.md

```diff
@@ -55,33 +55,32 @@ _Fill in purpose of this project_
 
 _Edit as necessary_
 
-Bitbucket pipelines need to be active and configured  
+CI pipelines need to be active and configured
 
 This project follows the following branch convention:
 **master** Main development branch
 **deployment** Deployment branch that gets tagged and tagged releases are deployed
```

Comment:
We should make sure to explicitly document that PRs should not be made into `deployment`, and that ideally projects should be setup with merge protection if possible to effectively reject any PRs into that branch.

Reply:
Added this https://github.com/ackama/silverstripe-cwp-template/issues/10